kadi/commands/tests/test_commands.py

```diff
@@ -865,10 +865,9 @@ def test_get_starcat_only_agasc1p8():
     """
     with (
         conf.set_temp("cache_starcats", False),
-        conf.set_temp("date_start_agasc1p8_earliest", "1994:001"),
-        conf.set_temp("date_start_agasc1p8_latest", "1994:002"),
+        conf.set_temp("date_start_agasc1p8", "1994:001"),
     ):
-        # Force AGASC 1.7 and show that star identification fails
+        # Force AGASC 1.8 and show that star identification fails
```

Comment:
since we are going back and forth with this test, @taldcroft, did you see this change in the comment?

Just making sure I did not change it wrongly.

Reply:
Yes, I saw this and agreed.